test/utils.py

```diff
@@ -55,7 +55,9 @@ def LONG_TESTS():
     # override LONG_TESTS with environment variable if available
     if os.getenv("LONG_TESTS"):
         LONG_TESTS = int(os.getenv("LONG_TESTS"))
-    return LONG_TESTS
+        if LONG_TESTS > 0:
+            return True
+        return False
```

Comment:
Here if we don't specify the `LONG_TESTS` environment variable the function will return `None`, which is falsy, but we most definitely only want it to be either `True` or `False`.